src/test/encoding/readable.sh

```diff
@@ -8,7 +10,7 @@ failed=0
 numtests=0
 pids=""
 
-myversion=`ceph-dencoder version`
+myversion=`./ceph-dencoder version`
```

Comment:
we set `PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}:${CMAKE_SOURCE_DIR}/src:$ENV{PATH}` for ceph tests in cmake. so instead of reverting the change in this script, can we have
```sh
if which ceph-dencoder &> /dev/null; then
  CEPH_DENCODER=ceph-dencoder
else
  CEPH_DENCODER=./ceph-dencoder
fi
```

and replace all occurrences of `ceph-dencoder` with "$CEPH_DENCODER"?